docs/development/deploying-locally/index.md

```diff
@@ -1,3 +1,50 @@
 # Deploying locally
 
 REANA can be easily deployed locally on your laptop using Minikube.  Useful for REANA platform development.
+
+### Instructions
+
+* (1) Clone the main REANA repository:
+```console
+$ git clone https://github.com/reanahub/reana.git
+$ cd reana
+```
+
+* (2) Create a REANA development setup as follows:
+```console
+$ GITHUB_USER=JOHNDOE make setup clone prefetch
+```
+
+* (3) Build all components:
+```console
+$ make build
+```
+
+
+* (4) Deploy REANA:
+```console
+$ make deploy
```

Comment:
We miss part what to do after deployment.  At least a sentence going like

Your REANA system is now deployed on your laptop.  You can launch example workflows like

```console
$ DEMO=reana-demo-helloworld make example
```